tools/LogCollector/source/omslinux_agentlog.py

```diff
@@ -141,6 +141,12 @@ def copyContainerFiles(omsContainerName, omsLinuxType):
     cmd='docker cp omsagent:' + '/var/opt/microsoft/omsconfig/omsconfig.log ' + '/tmp/omslogs/container'
     out=execCommand(cmd)
     writeLogCommand(cmd)
+    cmd='docker cp omsagent:' + '/var/opt/microsoft/scx/log/scx.log ' + '/tmp/omslogs/container'
```

Comment:
these changes are irrelevant to your code review , I am not sure why these are part of your PR?

Reply:
I messed up with the squash, some commits from the master branch has been included, I just modified 4 files I will revert my commits and perform one final commit. Sorry for the confusion.